src/sqs.rs

```diff
@@ -54,6 +56,82 @@ pub fn get_client(shared_config: &aws_config::Config) -> Result<Client> {
     Ok(Client::from_conf(builder.build()))
 }
 
+const BATCH_SIZE: usize = 10;
+
+/// Send message to a queue from a stream with concurrent invocations of `SendMessageBatch`.
+///
+/// This function retrieves items from the stream until it has exhausted. Any errors
+/// in the stream, or while processing the stream, are returned as soon as they are encountered.
+///
+/// If `concurrency` is `None` then message batches are sent sequentially. A `concurrency` value of
+/// zero, `Some(0)`, is not allowed.
```

Comment:
```suggestion
/// zero, `Some(0)`, is not allowed and will result in an error.
```

I think it's useful to be precise here, because "not allowed" could mean "panics" or "is undefined behaviour".